changelog.txt

```diff
@@ -7,6 +7,10 @@ TGUI 0.8.7  (TBD)
 - Added option to not replace existing widgets when loading widgets from file
 - Added isAnimationPlaying function to Widget (for the show and hide animations)
 - Fixed linking issues when compiling TGUI as a static library while dynamically linking SFML
+- Added UserData property to widgets
+- Changed method of storing names of widgets
+- Fixed adding the same widget to multiple containers
+- Added possibility to connect signals before creating widget
```

Comment:
You don't need to change the changelog, I will take care of it later. The "Added UserData property to widgets" isn't really accurate as it already existed for a while, it just couldn't be saved/loaded.

Reply:
Delete this commit?

Reply:
Yeah, the changes to this file should be removed.